apis/apiextensions/v1/composition_patches.go

```diff
@@ -23,7 +23,7 @@ import (
 
 	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
 
-	"github.com/crossplane/crossplane/pkg/validation/errors"
+	"github.com/crossplane/crossplane/internal/errors"
```

Comment:
Maybe `internal/validation/errors` for this package? That way its less likely to get confused with the existing general error-handling package in crossplane-runtime.

Reply:
moved 👍